addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -173,7 +173,7 @@
                             <field name="company_id" invisible="1" />
                             <field name="state" invisible="1" />
                             <field name="partner_id" options="{'no_open': True, 'no_create': True, 'no_edit': True}" invisible="not project_id"/>
-                            <field name="date_deadline" invisible="state in ['1_done', '1_canceled']"/>
+                            <field name="date_deadline" invisible="state in ['1_done', '1_canceled']" decoration-danger="date_deadline &lt; current_date"/>
```

Comment:
shouldn't it be if the deadline is after today?

Reply:
it's red when we exceed the deadline

Reply:
euh yes indeed. me dumdumb.